packages/frontend/core/src/components/workspace-selector/workspace-card/quota-check.tsx

```diff
@@ -0,0 +1,188 @@
+import { useConfirmModal } from '@affine/component';
```

Comment:
why put under workspace-selector folder?

Reply:
I haven't decided where to put it yet, do you have any suggestions?

Reply:
cloud workspace root

Reply:
https://github.com/toeverything/AFFiNE/blob/89638264636db9ec8874a2af51f0293e464851eb/packages/frontend/core/src/mobile/pages/workspace/layout.tsx#L127

https://github.com/toeverything/AFFiNE/blob/89638264636db9ec8874a2af51f0293e464851eb/packages/frontend/core/src/desktop/pages/workspace/layouts/workspace-layout.tsx#L30